nginx/tests/test.sh

```diff
@@ -1,33 +1,35 @@
 #!/bin/sh
+#/ Usage: test.sh <pkg_ident>
+#/
+#/ Example: test.sh core/php/7.2.8/20181108151533
+#/
 
-TESTDIR="$(dirname "${0}")"
-PLANDIR="$(dirname "${TESTDIR}")"
-SKIPBUILD=${SKIPBUILD:-0}
+set -euo pipefail
 
-hab pkg install --binlink core/bats
+if [[ -z "${1:-}" ]]; then
+  grep '^#/' < "${0}" | cut -c4-
+	exit 1
+fi
 
+TEST_PKG_IDENT="${1}"
+export TEST_PKG_IDENT
+hab pkg install core/bats --binlink
 hab pkg install core/busybox-static
-hab pkg binlink core/busybox-static ps
 hab pkg binlink core/busybox-static netstat
-hab pkg binlink core/busybox-static wc
-hab pkg binlink core/busybox-static uniq
-
-source "${PLANDIR}/plan.sh"
-# Unload the service if its already loaded.
-hab svc unload "${HAB_ORIGIN}/${pkg_name}"
-
-if [ "${SKIPBUILD}" -eq 0 ]; then
-  set -e
-  pushd "${PLANDIR}" > /dev/null
-  build
-  source results/last_build.env
-  hab pkg install --binlink --force "results/${pkg_artifact}"
-  hab svc load "${pkg_ident}"
-  popd > /dev/null
-  set +e
-
-  # Give some time for the service to start up
-  sleep 5
-fi
+hab pkg binlink core/busybox-static ps
```

Comment:
`ps` and `netstat` does not appear to be used in the tests. Can this be removed? Do we need `core/busybox-static` at all?

Reply:
Reference `test.bats` from this branch: https://github.com/predominant/core-plans/blob/predominant/nginx-1.17.1/nginx/tests/test.bats

`ps` is used on the following lines:

* Line 19
* Line 22
* Line 28
* Line 31

`netstat` is used on the following lines:

* Line 36